sdk/sdk-common/src/common/implementation/Price.ts

```diff
@@ -18,16 +18,16 @@ export class Price implements IPriceSerialized {
 
   readonly value: string
   readonly baseToken: Token
-  readonly quoteToken?: Token | CurrencySymbol
+  readonly quoteToken: Token | CurrencySymbol
 
   private constructor(params: IPriceSerialized) {
     this.value = params.value
     this.baseToken = params.baseToken
-    this.quoteToken = params.quoteToken ? params.quoteToken : Price.DEFAULT_QUOTE_TOKEN
+    this.quoteToken = params.quoteToken
   }
 
   static createFrom(params: { value: string; baseToken: Token; quoteToken?: Token }): Price {
-    return new Price(params)
+    return new Price({value: params.value, baseToken: params.baseToken, quoteToken: params.quoteToken || Price.DEFAULT_QUOTE_TOKEN})
   }
```

Comment:
I've moved the optionality, as in fact `quoteToken` is always defined due to the default value however the type says that it might be undefined which forces me to do checks on value that will always be there.